tools/lambda-promtail/lambda-promtail/promtail.go

```diff
@@ -126,9 +126,11 @@ func (b *batch) createPushRequest() (*logproto.PushRequest, int) {
 }
 
 func (b *batch) flushBatch(ctx context.Context) error {
-	err := b.client.sendToPromtail(ctx, b)
-	if err != nil {
-		return err
+	if b.client != nil {
```

Comment:
Probably uncalled for but this thing got crashed when I was fixing the failed unit tests. Made a nil check here :D